src/main.ts

```diff
@@ -145,8 +145,8 @@ async function migrateJsonGenerator() {
 
 const createSplashWindow = async () => {
   splashscreenWindow = new BrowserWindow({
-    width: 700,
-    height: 355,
+    width: 600,
+    height: 400,
```

Comment:
The new size has the same aspect ratio as the default one for the main window. I don't feel strongly about that, but I think it looks better with less white space on the sides